tutorials/users/catalog/gui-search.md

```diff
@@ -1 +1,115 @@
+3. The remainder of the page is laid out in a two-column format: on the left hand side you have information about the collection
+    itself, while on the right hand side you see a list of the items in the collection.
+
+    The left side contains various information about the collection in the following order:
+    ```{attention}
+    This section will be updated after changes to STAC.
+    ```    
+    - **Description** of the collection. Here it is "Coupled Model Intercomparison Project phase 6".
+    - **Keywords** associated with the collection. Here the keywords are 'CMIP', 'CMIP5', 'WCRP' and 'Climate Change'.
+    - **License** under which the data in the collection has been made available. Here the license is 'CC-BY-4.0'.
+    - **Temporal Extents** of the data. The information presented here shows that the data starts at year 1850 and goes until present day.
+    - **Spatial Extents** of the data are shown on the map. Here, we see two boxes that cover the whole earth, but with different longitude
+        ranges. This is because the CMIP6 data is global, but some models publish data in the latitudinal range -180 to 180, while others
+        publish in the range 0 to 360.
+    - **Metadata** section shows the various metadata associated with the collection. 
+    
+    The right side tiles the items in the collection. Each item tile shows the item's unique ID, its temporal extents, 
+    as well as the types of the asset files associated with the item.
+
+## Inspecting a collection item
+
+If you click on an item in the collection, you will be taken to a page about the item, which you can expect to look similar to 
+that shown in the image below:
+
+(item_view)=
+![Item View](images/cmip6_item.png)
+
+You will immediately recognize several UI elements of the page from discussions in earlier sections. In 
+particular, you will notice that the page is again organized in two columns, but unlike the collection view introduced in the last section, 
+this time the metadata about the item is actually presented on the right side. This metadata is sub-divided into several 
+categories; in the image above the categories seen are "General" and "Cmip6", but there is another category outside the view of the screenshot called "Data Cube". 
+There will always be a "General" section and what other sections you see for an item, will depend on the STAC extensions that have been applied to that item. 
+Here, the CMIP6 data uses the [cmip6](https://github.com/dchandan/stac-extension-cmip6) and the [Data Cube](https://github.com/stac-extensions/datacube) extensions, 
+so metadata described by those extensions are included within 
+their sections. The "General" section will always contain at least the temporal information about the item; any additional information 
+displayed will depend on what the creator of the catalog decided to include.
+
```

Comment:
Make the Data Cube details a separate paragraph and include a screenshot of the Data Cube.